JASP-Engine/JASP/R/common.R

```diff
@@ -687,6 +692,10 @@ callback <- function(results=NULL) {
 
 	if (is.na(value))
 		return("NaN")
+    
+    if (identical(value, numeric(0))) {
```

Comment:
Very minor: can you make this fit the rest of the function (indent & curly braces)?